vimmo/db/run_updates.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Find the directory where this script is located --> $0 is script being run, realpath gets the absolute path
+script_dir=$(dirname "$(realpath "$0")")
```

Comment:
not sure how this works but if this is better just simply use it for the cron_job.sh